source/_integrations/trafikverket_weatherstation.markdown

```diff
@@ -14,75 +15,25 @@ ha_platforms:
 
 Showing weather information provided by [Trafikverket](https://www.trafikverket.se/) weather stations in Sweden.
 
-Potential use cases:
+## Potential use cases:
 
 - Get weather data in general.
 - You live near a weather station and want to know the current weather conditions at home.
 - Setup automations for your car heating system. If the road is frozen along the way to work, you might want the car heating system to start earlier.
 
-## Configuration
+{% include integrations/config_flow.md %}
 
-To enable this sensor, add the following lines to your `configuration.yaml`.
-
-```yaml
-sensor:
-  - platform: trafikverket_weatherstation
-    name: Trafikverket Kungälv
-    api_key: YOUR_API_KEY
-    station: Kungälv
-    monitored_conditions:
-        - air_temp
-        - road_temp
-        - humidity
-        - precipitation
-        - wind_direction
-        - wind_direction_text
-        - wind_speed
-        - wind_speed_max
-        - precipitation_amount
-        - precipitation_amountname
-```
-
-{% configuration %}
-name:
-  description: Your unique name of the device in the frontend.
-  required: true
-  type: string
-api_key:
-  description: Your personal API key from Trafikverket.
-  required: true
-  type: string
-station:
-  description: Name of the weather station from Trafikverket.
-  required: true
-  type: string
-monitored_conditions:
-  description: Specify what measurement data to retrieve from the weather station.
-  required: true
-  type: list
-  keys:
-    air_temp:
-      description: Air temperature.
-    road_temp:
-      description: The temperature in the road.
-    humidity:
-      description: Relative humidity.
-    precipitation:
-      description: Type of precipitation (Swedish text).
-    wind_direction:
-      description: Wind direction in degrees.
-    wind_direction_text:
-      description: Rough wind direction in twelve variants (Swedish text).
-    wind_speed:
-      description: Average wind speed during the last 10 minutes.
-    wind_speed_max:
-      description: Maximum wind speed measured during the last 30 minutes.
-    precipitation_amount:
-      description: Amount of precipitation.
-    precipitation_amountname:
-      description: Amount of precipitation in thirteen variants (Swedish text).
-    
-{% endconfiguration %}
+## Entities provided by the integration is
+- Air temperature.
+- Road temperature.
+- Relative humidity.
+- Type of precipitation (Swedish text).
+- Wind direction in degrees.
+- Rough wind direction in twelve variants (Swedish text).
+- Average wind speed during the last 10 minutes.
+- Maximum wind speed measured during the last 30 minutes.
+- Amount of precipitation.
+- Amount of precipitation in thirteen variants (Swedish text).
 
 ## Obtaining API key
```

Comment:
Let move this above the config flow setup instructions (as one would need it before doing that); and lets name the chapter "Prerequisites" to match other integrations.